content/ADR-133-scene-runtime.md

```diff
@@ -69,7 +69,6 @@ The runtime for the SDK7 is compatible with [CommonJS](https://en.wikipedia.org/
 
 The exposed RPC modules are defined in the [protocol repository](https://github.com/decentraland/protocol/blob/main/proto/decentraland/kernel/apis/engine_api.proto).
 
-> TODO: define and document naming conventions about code generation for modules
```

Comment:
The naming and code generation is controlled by the protocol repo for **not only the scene runtime** (this ADR)